docs/products/kafka/kafka-connect/reference/s3-sink-additional-parameters.rst

```diff
@@ -0,0 +1,49 @@
+S3 sink connector naming and data formats
+=========================================
+
+The Apache Kafka Connect® S3 sink connector enables you to move data from an Aiven for Apache Kafka cluster to Amazon S3 for long term storage. The following document describes advanced parameters defining the naming and data formats.
+
+
+S3 naming format
+---------------- 
+
+The Apache Kafka Connect® S3 sink connector by Aiven stores a series of files in the specified S3 bucket. By default, each object is named using the pattern:
```

Comment:
Maybe `stores a series of files as objects`? Otherwise it doesn't explain how objects and files are related.